docs/src/examples/components/List/Types/index.tsx

```diff
@@ -14,6 +14,11 @@ const Types = () => (
       description="A list can be formatted to indicate that its items can be selected."
       examplePath="components/List/Types/ListExampleSelection"
     />
+    <ComponentExample
+      title="Active"
+      description="A list can be formatted to indicate that one or more items are active."
```

Comment:
as `active` mainly is an attribute of `ListItem`, it would be better to provide description like `A list item can be formatted to indicate its active state.` - this would better reflect semantics that stays behind